index.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+</head>
+<body>
+    <!-- Navigation Bar-->
+    <header class="header-container">
+        <div class="left">
+            <a href="/blank"><img id="icons" src="./graphics/open-menu.png" alt="menu"></a>
+            <a href="https://www.youtube.com/"><img class="logo" src="graphics/yt_logo_rgb_light.png" alt="Main-logo"></a>
+
+        </div>
+        <div class="center">
+            <input type="text">
+            <button><img id="icons" src="./graphics/icons8-search-50.png" alt="Search"></button>
+        </div>
+        <div class="right">
+            <a href="/blank"><img id="icons" src="./graphics/upload.png" alt="upload"></a>
+            <button class="sign-in">Sign In</button>
+        </div>
```

Comment:
- [ ] It needs to have at least 3 images on the right part of the navbar, as specified on [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/)
![image](https://user-images.githubusercontent.com/53256139/86982503-07766c00-c14f-11ea-8f72-9923abe53507.png)